Cargo.toml

```diff
@@ -12,6 +12,7 @@ members = [
   "net_connection",
   "net_ipc",
   "sodium",
+  "hc-dpki",
```

Comment:
a strange detail to request, but nevertheless, would you be able to rename the crate "hc_dkpi" instead of with a dash, to match all the other ones (except one which should also be changed)